.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "modules/iv"]
+	path = external/iv
+	url = https://github.com/BlueBrain/iv.git
```

Comment:
I am using BlueBrain submodule because PR is not merged yet: https://github.com/neuronsimulator/iv/pull/10

Reply:
Pull request now merged in neuronsimulator/iv